src/utils.js

```diff
@@ -11,7 +11,7 @@ import { PACKAGE_TYPES, LOCAL_PROTOCOLS } from 'const';
 
 /* global nodeRequire, localesRoot */
 
-const SOURCE_MAP_RE = new RegExp(/\/\/[#@]\s(source(?:Mapping)?URL)=\s*(\S+)/);
+const SOURCE_MAP_RE = /\/\/[#@]\s(source(?:Mapping)?URL)=\s*(\S+)/;
```

Comment:
My understanding is that `new RegExp` was useless here, its only argument is already a regexp.